.github/workflows/run_codecov_on_pull_request.yml

```diff
@@ -0,0 +1,40 @@
+name: codecov.yml
+# Run Codecov on PR open or change
+on: pull_request
+jobs:
+  codecov:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Check out this repository code
+        uses: actions/checkout@v3
+        with:
+          path: empower
+          fetch-depth: 0
+
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18"
+
+      - name: Run React tests
+        run: |
+          npm install
+          npm test -- --coverage
+        working-directory: ./empower/react
+
+      - name: Run Python Tests
+        run: |
+          pip install -U pytest
+          pip install pytz
+          pip install pytest-codecov
+          pytest --cov=. --cov-report=xml
+        working-directory: ./empower/flask/src
+
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v3
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          flags: react, flask
```

Comment:
- Need to upload reports with flags separately. Something like:

```
- name: Upload React coverage reports to Codecov
        uses: codecov/codecov-action@v3
        with:
          token: ${{ secrets.CODECOV_TOKEN }}
          flags: react
```

```
- name: Upload Flask coverage reports to Codecov
        uses: codecov/codecov-action@v3
        with:
          token: ${{ secrets.CODECOV_TOKEN }}
          flags: flask
```

- Need to add additional config that has something to do with `path` (or may be some other similar name, prob passed under `with` key). Neil suggests it may be `file` or `files` key

Reply:
Plan to address this one before merging in 👍